Source/ConversationV1/Models/EntityCollection.swift

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright IBM Corporation 2017
+ * Copyright IBM Corporation 2018
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
```

Comment:
All of the collection models may be affected by the pagination changes that the Conversation team are planning to do.  Hopefully this won't require any work on our part except to regen the service, but its something to keep on your radar.

Reply:
Thanks for the reminder! Yeah, we should regenerate the service after their updates.